app/src/services/chesSvc.js

```diff
@@ -40,49 +40,98 @@ class ChesService {
   /** @function dataService
    *  Gets the current DataService
    */
-  get dataService () {
+  get dataService() {
     return this._dataService;
   }
 
   /** @function dataService
    *  Sets the current DataService
    *  @param {object} v - a DataService object.
    */
-  set dataService (v) {
+  set dataService(v) {
     this._dataService = v;
   }
 
   /** @function emailService
    *  Gets the current EmailService
    */
-  get emailService () {
+  get emailService() {
     return this._emailService;
   }
 
   /** @function emailService
    *  Sets the current EmailService
    *  @param {object} v - am EmailService object.
    */
-  set emailService (v) {
+  set emailService(v) {
     this._emailService = v;
   }
 
   /** @function queueService
    *  Gets the current QueueService
    */
-  get queueService () {
+  get queueService() {
     return this._queueService;
   }
 
   /** @function queueService
    *  Sets the current QueueService
    *  @param {object} v - a QueueService.
    */
-  set queueService (v) {
+  set queueService(v) {
     this._queueService = v;
   }
 
-  async getStatus (client, messageId) {
+  /** @function findStatuses
+   *  @description Finds the set of message statuses that matches the search criteria
+   *
+   *  @param {string} client - the authorized party / client
+   *  @param {string} messageId - the id of the desired message
+   *  @param {string} status - the desired status of the messages
+   *  @param {string} tag - the desired tag of the messages
+   *  @param {string} transactionId - the id of the desired transaction
+   *  @param {string} fields - a comma separated list of desired columns in addition to the defaults
+   *  @throws Problem if an unexpected error occurs
+   *  @returns {object[]} Array of Status objects with a subset of properties
+   */
+  async findStatuses(client, messageId, status, tag, transactionId, fields) {
+    let fieldArray;
+    if (fields) {
```

Comment:
The transformer should house all functional/static object shape conversions. While the fields could be considered a shape change in their own right, that block is only really renaming attributes of an in-transit object. Since the source and destination objects do not actually represent API nor DB Model objects, I don't think it should be in the transformer.

This though is a completely null point however as we decided to not use the fields parameter anymore anyways.